actions/admin/paymail_addresses.go

```diff
@@ -130,6 +131,15 @@ func paymailCreateAddress(c *gin.Context, _ *reqctx.AdminContext) {
 		opts = append(opts, engine.WithMetadatas(requestBody.Metadata))
 	}
 
+	config := reqctx.AppConfig(c)
```

Comment:
IMHO PR is a good opportunity to write some endpoint tests for create paymail address endpoint.